apps/examples/lcd_test/example_lcd.c

```diff
@@ -254,13 +254,105 @@ static void test_bit_map(void)
 	free(lcd_data);
 }
 
+static void test_fps(void){
```

Comment:
I left this comment third times.
This line has a coding rule violation. Please fix it.
```
static void test_fps(void){
-->
static void test_fps(void)
{
```
`{` should be located at next line for function.

Reply:
done